web-api/src/storage.rs

```diff
@@ -268,6 +268,11 @@ pub(crate) trait Size {
 
 #[async_trait(?Send)]
 pub(crate) trait IndexedProperties {
+    async fn cached_schema(&self) -> Result<IndexedPropertiesSchema, Error> {
```

Comment:
❓ since caching apparently won't be implemented in the near future, should we remove this trait function as it currently only acts as a wrapper for the `load_schema()`? otherwise one could get the false impression when not looking close enough that some kind of caching is already implemented. we can always reintroduce the function once/if caching will be implemented.